src/components/DeckMechanisms/index.js

```diff
@@ -423,7 +420,7 @@ export default class DeckMechanisms extends React.Component {
     this.setState({
       playerOrder,
       turn,
-      mana: DEFAULT_MANA + (turn - 1),
```

Comment:
I'm not sure if this is used anywhere else on the site, but I'd think that the second played has more mana than the first - 4 on turn 1 for example